packages/core/src/store/feed-aggregation-store.ts

```diff
@@ -69,9 +81,41 @@ export class FeedAggregationStore extends ObjectStore<number, StreamID> {
     }
   }
 
+  streamIDs(gt?: string): ReadableStream<[string, StreamID]> {
+    const source = new StreamIDFeedSource(this.find, this.onWrite.asObservable(), gt)
+    return new ReadableStream(source)
+  }
+
   async close(): Promise<void> {
     clearInterval(this.#interval)
     await this.tasks.onIdle()
     await super.close()
   }
 }
+
+class StreamIDFeedSource implements UnderlyingSource<[string, StreamID]> {
+  /**
+   * Opaque token that stores current position in the feed. Used as a resumability token.
+   */
+  private token: string
+
+  constructor(
+    private readonly find: FeedAggregationStore['find'],
+    private readonly onWrite: Observable<void>,
+    token: string = process.hrtime.bigint().toString()
+  ) {
+    this.token = token
+  }
+
+  async pull(controller: ReadableStreamController<[string, StreamID]>) {
+    const entries = await this.find({ limit: controller.desiredSize, gt: this.token })
```

Comment:
concrete proposal is to change this `gt` to `gte`.

And possibly also rename "after" to "at" or "resumeAt" (which is what rust-ceramic uses) throughout the APIs to match the updated semantics

Reply:
> concrete proposal is to change this `gt` to `gte`.
> 
> 
> 
> And possibly also rename "after" to "at" or "resumeAt" (which is what rust-ceramic uses) throughout the APIs to match the updated semantics

Consider the db containing a single entry A. Just changing GT to GTE would mean an infinite stream of As, which seems like an abuse of "at least once" semantics.

Reply:
Only if the feed connection was stopped and re-created with a resumeToken right?  If the feed is established and remains open it should just return the single document and then wait for new data right?

Yeah definitely agreed we shouldn't return the same stream over and over for a single feed session, my concern is when the feed is shut down for any reason (transient network error, js-ceramic restart, client restart, etc) and then resumed, I want to make sure there's no chance of missing updates.

Reply:
> Only if the feed connection was stopped and re-created with a resumeToken right?  If the feed is established and remains open it should just return the single document and then wait for new data right?
> 
> 
> 
> Yeah definitely agreed we shouldn't return the same stream over and over for a single feed session, my concern is when the feed is shut down for any reason (transient network error, js-ceramic restart, client restart, etc) and then resumed, I want to make sure there's no chance of missing updates.

So, without a better proposal, do you expect me to play telepathy?